lib/classes/PluginManager.js

```diff
@@ -309,6 +310,41 @@ class PluginManager {
     });
   }
 
+  loadVariableResolvers(pluginInstance) {
+    const pluginName = pluginInstance.constructor.name;
+    for (const [variablePrefix, resolverOrOptions] of _.entries(
+      pluginInstance.variableResolvers || {}
+    )) {
+      let options = resolverOrOptions;
```

Comment:
I would silently discard (do not process) here falsy values (if e.g. someone decides to clear `foo` resolver with `foo: null`, we should handle that gently)

Reply:
> if they want to do that, they should delete this.variableResolvers.foo or never set it in the first place.

When cleanness of code is concerned I totally agree. Still when we speak of validation, imo we should not be that restrictive, and simply skip on `foo: null`. Throwing with message  that _optional_ variable resolver was set to e.g. `null` would be quite weird.

If something is optional, convention is rather to ignore empty values, and not to unconditionally expect them to be undefined.